cron/cron.go

```diff
@@ -56,13 +56,19 @@ func (ct *Crontab) AddJob(id string, labels map[string]string, jobType string) e
 	default:
 		logrus.Warnf("Unknown job type: %s", jobType)
 	}
-
-	return ct.cronRunner.AddJob(schedule, job)
+	err := ct.cronRunner.AddJob(schedule, job)
+	if err != nil {
+		logrus.Errorf("error adding: %s. Got: %s", id, err)
+	} else {
```

Comment:
if you log the error and return it, you should be all set. You won't need the else. Just log info and return nil.